spring-rabbit/src/main/java/org/springframework/amqp/rabbit/log4j2/AmqpAppender.java

```diff
@@ -144,7 +146,7 @@ public AmqpAppender(String name, Filter filter, Layout<? extends Serializable> l
 		this.events = eventQueue;
 	}
 
-	@PluginFactory
```

Comment:
So, if we are going to keep this annotation, a conflict will happen and Log4J won't know which option to choose for configuration?

I don't mind to remove it since a builder approach is fully compatible with Log4J inversion of control; just want some clarification for myself.

Thanks

Reply:
"When plugins are being constructed after a configuration has been parsed, a plugin builder will be used if available, otherwise a plugin factory method will be used as a fallback. If a plugin contains neither factory, then it cannot be used from a configuration file (it can still be used programmatically of course)." 

I will restore it